src/main/java/org/broadinstitute/dsm/jobs/TestBostonUPSTrackingJob.java

```diff
@@ -508,6 +543,43 @@ private void updateTrackingInfo(PoolingDataSource<PoolableConnection> dataSource
 
     }
 
+    /**
+     * Queries the database to see whether there is an upload reason
+     * for the given kit label
+     */
+    private boolean doesKitHaveUploadReason(Connection conn, String kitLabel) throws SQLException {
+        boolean hasUploadReason = false;
+        try (PreparedStatement stmt = conn.prepareStatement(SQL_GET_KIT_REQUEST + BY_DDP_LABEL)) {
```

Comment:
I decided to use existing queries for this--if there's risk that such a query might pick up non-tbos orders, please let me know and we can go by `dsmKitRequestId` or something else.

Reply:
I think instead of this query, you want to use `SELECT * FROM FROM ddp_kit_request req LEFT JOIN ddp_kit kit on (kit.dsm_kit_request_id = req.dsm_kit_request_id) WHERE kit.kit_label = ?`
the `ddp_label` is null in test boston